client/src/legacy/LeftAndMain.js

```diff
@@ -1076,7 +1089,9 @@ $.entwine('ss', function($) {
       let root = this.getReactRoot();
       if (!root) {
         root = createRoot(this[0]);
+        this.setReactRoot(root);
```

Comment:
Not setting the property, and not adding it in line 1057 in the first place, are ultimately the root cause of this bug.